app/controllers/admin/base_controller.rb

```diff
@@ -17,6 +18,10 @@ def set_cache_headers
       response.cache_control.replace(private: true, no_store: true)
     end
 
+    def set_referrer_policy_header
+      response.headers['Referrer-Policy'] = 'same-origin'
+    end
```

Comment:
Does this not override the setting from `config/environments/production.rb`?

Reply:
It should, for the admin UI.